map/autoware_map_projection_loader/src/map_projection_loader.cpp

```diff
@@ -25,39 +25,50 @@
 
 namespace autoware::map_projection_loader
 {
-tier4_map_msgs::msg::MapProjectorInfo load_info_from_yaml(const std::string & filename)
+autoware_map_msgs::msg::MapProjectorInfo load_info_from_yaml(const std::string & filename)
 {
   YAML::Node data = YAML::LoadFile(filename);
 
-  tier4_map_msgs::msg::MapProjectorInfo msg;
+  autoware_map_msgs::msg::MapProjectorInfo msg;
   msg.projector_type = data["projector_type"].as<std::string>();
-  if (msg.projector_type == tier4_map_msgs::msg::MapProjectorInfo::MGRS) {
+  if (msg.projector_type == autoware_map_msgs::msg::MapProjectorInfo::MGRS) {
     msg.vertical_datum = data["vertical_datum"].as<std::string>();
     msg.mgrs_grid = data["mgrs_grid"].as<std::string>();
 
   } else if (
-    msg.projector_type == tier4_map_msgs::msg::MapProjectorInfo::LOCAL_CARTESIAN_UTM ||
-    msg.projector_type == tier4_map_msgs::msg::MapProjectorInfo::TRANSVERSE_MERCATOR) {
+    msg.projector_type == autoware_map_msgs::msg::MapProjectorInfo::LOCAL_CARTESIAN_UTM ||
+    msg.projector_type == autoware_map_msgs::msg::MapProjectorInfo::TRANSVERSE_MERCATOR) {
     msg.vertical_datum = data["vertical_datum"].as<std::string>();
     msg.map_origin.latitude = data["map_origin"]["latitude"].as<double>();
     msg.map_origin.longitude = data["map_origin"]["longitude"].as<double>();
     msg.map_origin.altitude = data["map_origin"]["altitude"].as<double>();
 
-  } else if (msg.projector_type == tier4_map_msgs::msg::MapProjectorInfo::LOCAL) {
+  } else if (msg.projector_type == autoware_map_msgs::msg::MapProjectorInfo::LOCAL) {
     ;  // do nothing
 
+  } else if (msg.projector_type == "local") {
+    RCLCPP_WARN_STREAM(
+      rclcpp::get_logger("MapProjectionLoader"),
+      "Load " << filename << std::endl
```

Comment:
```suggestion
      "Load " << filename << "\n"
```

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/9392/commits/81625e3a7e39f45068b623e7b63207fe7b575554